ui/app/AppLayouts/CommunitiesPortal/stores/CommunitiesStore.qml

```diff
@@ -85,6 +88,16 @@ QtObject {
         root.communitiesModuleInst.importCommunity(communityKey);
     }
 
+    function requestCommunityInfo(communityKey) {
+        let publicKey = communityKey
+        if (globalUtils.isCompressedPubKey(communityKey)) {
```

Comment:
This is problematic (hard to mock/test), accessing naked context properties. Suggest using functions from Utils

Reply:
I tried `Utils` but it's not available in this store. Will look into how it can be imported

Reply:
Yeah, need to import it